packages/config/src/options.ts

```diff
@@ -198,6 +198,12 @@ const driverConfigOptions: Array<DriverConfigOption> = [
     defaultValue: 60000,
     validation: validate.isNumber,
     overrideLevel: 'any',
+  }, {
+    name: 'experimentalCspAllowList',
+    defaultValue: false,
+    validation: validate.validateAny(validate.isBoolean, validate.isSubsetOf('script-src-elem', 'script-src', 'default-src', 'sandbox', 'form-action', 'navigate-to')),
```

Comment:
isSubsetOf to me indicates you could specify all of these options. Can we rename to reflect the intent?

Reply:
@emilyrohrbough You can specify all those options.

Reply:
@emilyrohrbough What would you suggest as a name?
`includesOnly`
`matchEvery`
`membersOf`